### DISCUSSION TITLE
Starstruck achievement requirement makes no sense

### DISCUSSION THREAD

**Discussion by minusQuantumNeko** [community/community#40799] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I find current achievement requirement really weird, as it requires you to be the original repository creator (from what I know)! But there are a lot of problems with such requirement, for example: Someone creates a repository for your program (organization owner most likely), and you are the only developer of it. Your program gets popular and receives many stars, but you don't get any achievements, just because you are not original creator!

**Comment by ossanna16** [comment]
Hi there @minusQuantumNeko and welcome to our community! Thank you for the feedback!
